lib/dest/writeContents/utils.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var fs = require('graceful-fs');
+
+function utimes(writePath, stat, cb) {
+  if (stat.mtime) {
+    stat.atime = stat.atime || new Date();
```

Comment:
This add a new `atime` equals to current time if no one is provided. This is done this way because `fs.utimes()` need both `atime` and `mtime`.